internal/dbmodel/group_test.go

```diff
@@ -36,3 +36,19 @@ func TestGroupEntry(t *testing.T) {
 	c.Assert(result.Error, qt.IsNil)
 	c.Assert(ge3, qt.DeepEquals, ge)
 }
+
+func TestDeleteGroupEntry(t *testing.T) {
```

Comment:
TestHardDeleteGroupEntry is a better name. Maybe even a comment too explaining why we're doing this test would be good.